vulnerabilities/importer_yielder.py

```diff
@@ -129,7 +129,7 @@
     },
     {
         'name': 'redhat',
-        'license': '',
+        'license': 'CC BY 4.0',
```

Comment:
Please use SPDX license indentifiers OR scancode license identifiers.... But do not make something new.